plural/terraform/aws-cluster-api/main.tf

```diff
@@ -0,0 +1,59 @@
+resource "kubernetes_namespace" "console" {
```

Comment:
why can't we just use the existing console terraform modules?  Aren't they effectively the same?

Reply:
we probably could, but we still need `deps.yaml` that targets `aws-bootstrap-cluster-api`. Is it possible to use new deps and link existing files?